core/templates/simple_layout.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
Is this file a little redundant? Could we just remove some of the options on the regular layout?